.github/workflows/tests-e2e-npm.yml

```diff
@@ -5,23 +5,27 @@ on:
       env_vars:
         required: false
         type: string
-        default: '{}'
+        default: "{}"
       npm_build_command:
         required: false
         type: string
-        default: ''
+        default: ""
       pre_test_command:
         required: false
         type: string
-        default: ''
+        default: ""
       docker_compose_file:
         required: false
         type: string
-        default: docker-compose.yml
+        default: ""
```

Comment:
please leave the default as `docker-compose.yml` and then we can set to `""`. Otherwise this will be really annoying to set in most places